agent/provider/readme.md

```diff
@@ -3,12 +3,12 @@
 ## Central setup
 We have centrally deployed (ip: `34.244.4.185`) three independent standalone modules/apps:
  - [net Mk1](https://github.com/golemfactory/yagna/blob/master/docs/net-api/net-mk1-hub.md) @ 34.244.4.185:7464 \
-   (can be invoked locally with `cargo run --release --example ya_sb_router`)
+   (can be invoked locally with `cargo run --release -p ya-sb-router --example ya_sb_router`)
```

Comment:
What does it do? The paragraph starts with saying there are deployed modules/apps. What does this do? Is it a client?

Reply:
this paragraph is to give context to the reader. You do not need to instantiate those. They are centrally deployed for all. Does this make sense?

Reply:
I think so. The term "invoked locally" is misleading, because that means, you're running this command or your computer, which is true, but misses the point. I think "can be run locally" would be better.

Reply:
ok, I've moved this section to the bottom, end reworded as you suggested. take a look

Reply:
You reworded 1 of 3, but it's going in the right direction ;)

Reply:
done